### PR TITLE
Issue 82: Afford starting any workflow manually

### PR DESCRIPTION
**Pull request by wwelling**

### CODE REVIEW

File: service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java [hunk 1]
```diff
@@ -52,15 +55,41 @@ public Workflow deactivate(String workflowId, String tenant, String token) throw
...
+    String definitionsUrl = String.format("%s/camunda/process-definition?deploymentId=%s", okapiUrl, deploymentId);
+    ResponseEntity<ArrayNode> response = this.restTemplate.getForEntity(definitionsUrl, ArrayNode.class);
+
+    if (response.getStatusCode() == HttpStatus.OK && !response.getBody().isEmpty()) {
+      logger.debug("Response body: {}", response.getBody());
+
+      try {
+        return response.getBody().get(0);
+      } catch (Exception e) {
+
```
Comment (kaladay) [inline service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java:82]: Should this have a `logger.debug()` call in the exception?

File: service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java [hunk 2]
```diff
@@ -74,15 +103,23 @@ private Workflow sendRequest(Workflow workflow, String requestPath, String tenan
         logger.info("Workflow is actvie = {}, deploymentID = {}", responseWorkflow.isActive(), deploymentId);
         return workflowRepo.save(responseWorkflow);
       } catch (Exception e) {
-        throw new WorkflowEngineServiceException("Unable to get updated workflow from workflow engine!");
+
```
Comment (kaladay) [inline service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java:106]: Should this have a `logger.debug()` call in the exception?

File: service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java (hunk 1 above)
Comment (wwelling) [reply service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java:82]: Nah, the debug log is handled in the controller advise.

File: service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java (hunk 2 above)
Comment (wwelling) [reply service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java:106]: https://github.com/TAMULib/mod-workflow/blob/0b23024970be2453c72ee1d8f98baf602d97c351/service/src/main/java/org/folio/rest/workflow/controller/advice/WorkflowControllerAdvice.java#L26

**Review by kaladay — APPROVED** [review]

**Pull request merged**